### ISSUE TITLE
How to use vue-upload-component with Nuxt ? 

### ISSUE THREAD

**Issue by duythang** [lian-yue/vue-upload-component#213]
Hello 

How can i setup vue-upload-component for nuxt ?. I followed  [link](https://lian-yue.github.io/vue-upload-component/#/en/documents#getting-started-ssr-server-isomorphism)
```
<template>
  <file-upload v-model="files" post-action="/">Upload file</file-upload>
</template>
<style>
/*
import '~vue-upload-component/dist/vue-upload-component.part.css'


or


 */
.file-uploads {
  overflow: hidden;
  position: relative;
  text-align: center;
  display: inline-block;
}
.file-uploads.file-uploads-html4 input[type="file"] {
  opacity: 0;
  font-size: 20em;
  z-index: 1;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  position: absolute;
  width: 100%;
  height: 100%;
}
.file-uploads.file-uploads-html5 input[type="file"] {
  overflow: hidden;
  position: fixed;
  width: 1px;
  height: 1px;
  z-index: -1;
  opacity: 0;
}
</style>
<script>
import FileUpload from 'vue-upload-component/dist/vue-upload-component.part.js'
export default {
  components: {
    FileUpload,
  },
  data() {
    return {
      files: []
    }
  },
}
</script>
```
I got issue 
```
_vm._ssrNode is not a function
```

**Comment by madc0w** [comment]
I don't get it.  How can this be closed?  Was it answered?

**Comment by blackfyre** [comment]
> I don't get it. How can this be closed? Was it answered?

It was closed with the commit 6448720
The commit itself is the answer, which should resolve this issue.
